sources/web/datalab/polymer/components/datalab-toolbar/datalab-toolbar.html

```diff
@@ -0,0 +1,119 @@
+<link rel="import" href="../../components/shared-styles/shared-styles.html">
+
+<link rel="import" href="../../bower_components/app-layout/app-header/app-header.html">
+<link rel="import" href="../../bower_components/app-layout/app-toolbar/app-toolbar.html">
+<link rel="import" href="../../bower_components/iron-icons/iron-icons.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../../bower_components/paper-dialog/paper-dialog.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+
+<dom-module id="datalab-toolbar">
+  <template>
+    <style include="datalab-shared-styles">
+      :host {
```

Comment:
It's minor. Could become more important later if more are added. Using a sort order of "sorted by implementation date" is generally not useful. If they are sorted according to where they appear on the screen, that's OK. If that's the case, a comment might be useful.